### PR TITLE
ci(action): auto-bump daily and update who notify to

### PR DESCRIPTION
**Pull request by v1v**
Run more often and at night, so there is no 1 week gap between golang releases and the golang-autobump
Notify in the `#ingest-notifications` channel.

### Question

- [x] What team shall be pinged in slack?

### CODE REVIEW

File: .github/workflows/bump-golang.yml [hunk 1]
```diff
@@ -21,3 +24,5 @@ jobs:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          notifySlackChannel: "#ingest-notifications"
+          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@update-me-with-the-slack-team-to-be-poked` please look what's going on <${{ env.JOB_URL }}|here>"
```
[the remark this answers is not in the document]
Comment (v1v) [reply .github/workflows/bump-golang.yml:28]: For instance, currently,  we are the ones receiving those notifications in Slack:

image


In this PR the ping will happen to `@WHO` - an unknown user/team in Slack. @jlind23, would you like to enable a specific ping in Slack? If so, what's the team name in Slack?

**Comment by v1v** [comment]
@elastic/ingest-eng-prod , can you please review this PR? 🙏

**Comment by alexsapran** [comment]
Hi Victor, 

If I understand the ask is what team inside ingest should be notified in case there is a problem with the golang-crossbuild images. I wanted to ack and provide my pov.

My take is the teams dependent on this library should be aware that there is a problem and get notified and try to fix it. 

@jlind23 and @roaksoax can comment on which specific teams this should be as I might be wrong and that team is the ingest-eng-prod.

**Comment by jlind23** [comment]
@pierrehilbert and the elastic agent team should be the one notified!

**Comment by v1v** [comment]
> @pierrehilbert and the elastic agent team should be the one notified!

Excellent, what's the slack team?

File: .github/workflows/bump-golang-previous.yml [hunk 2]
```diff
@@ -23,3 +26,5 @@ jobs:
           vaultUrl: ${{ secrets.VAULT_ADDR }}
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
+          notifySlackChannel: "#ingest-notifications"
+          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@update-me-with-the-slack-team-to-be-poked` please look what's going on <${{ env.JOB_URL }}|here>"
```
Comment (v1v) [inline .github/workflows/bump-golang-previous.yml:30]:
```suggestion
          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, @agent-team please look what's going on <${{ env.JOB_URL }}|here>"
```

**Comment by pierrehilbert** [comment]
@agent-team

**Review by jlind23 — APPROVED** [review]

**Comment by v1v** [comment]
@mergifyio backport 1.20

**Pull request merged**